### PR TITLE
Switch back to the openshift docker fork

### PR DESCRIPTION
**Pull request by nalind**
Switch from nalind's fork of github.com/moby/moby to openshift's.  Update github.com/containers/storage to v1.11.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -4,9 +4,8 @@ import:
   subpackages:
   - reference
 - package: github.com/docker/docker
-  repo:    https://github.com/nalind/docker.git
-  version: master
-
+  repo:    https://github.com/openshift/moby-moby
+  version: openshift-3.10-docker-b68221c
```
Comment (bparees) [inline glide.yaml]: is there a reason not to move up to a newer commit?  looks like moby-moby has moved significantly beyond this level.

Comment (nalind) [reply glide.yaml]: None specifically, but I thought this was the version we were using before we switched away from the openshift fork.

Comment (bparees) [reply glide.yaml]: (it seems like we should create a 4.0 branch in moby-moby and add the carries to it)

Comment (bparees) [reply glide.yaml]: we can merge this as a good first step but it probably makes sense to create a 4.0 branch at some point and move to that.

Comment (nalind) [reply glide.yaml]: Sounds good to me.

**Pull request merged**